File: pocket-android-eth/src/main/java/network/pokt/eth/util/RawFileUtil.java
package network.pokt.eth.util;

import android.content.Context;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStream;
import java.io.InputStreamReader;

public class RawFileUtil {

    public static String readRawTextFile(Context context, int resId) {
        InputStream inputStream;
        try {
            inputStream = context.getResources().openRawResource(resId);
        } catch (Exception exception) {
            return null;
        }

        InputStreamReader inputStreamReader = new InputStreamReader(inputStream);
        BufferedReader buffReader = new BufferedReader(inputStreamReader);
        String line;
        StringBuilder text = new StringBuilder();
        try {
            while (( line = buffReader.readLine()) != null) {
                text.append(line);
                text.append('\n');
            }
        } catch (IOException e) {
            return null;
        }
        return text.toString();
    }
}

File: pocket-android-aion/src/main/java/network/pokt/pocketaion/util/RawFileUtil.kt
package network.pokt.pocketaion.util

import android.content.Context
import java.io.BufferedReader
import java.io.IOException
import java.io.InputStream
import java.io.InputStreamReader

class RawFileUtil {

    companion object {
        fun readRawTextFile(context: Context, resId: Int): String? {
            val inputStream: InputStream
            try {
                inputStream = context.resources.openRawResource(resId)
            } catch (exception: Exception) {
                return null
            }

            val inputStreamReader = InputStreamReader(inputStream)
            val buffReader = BufferedReader(inputStreamReader)
            var line: String
            val text = StringBuilder()
            try {
                line = buffReader.readLine()
                while (line != null) {
                    text.append(line)
                    text.append('\n')
                }
            } catch (e: IOException) {
                return null
            }

            return text.toString()
        }
    }
}